yiisoft/yii2-bootstrap/CVE-2015-3397.yaml

```diff
@@ -0,0 +1,8 @@
+title:     JSON Data encoded for use in HTML was not safe to use in IE6/IE7, possible XSS attacks
+link:      http://www.yiiframework.com/news/86/yii-2-0-4-is-released/
+cve:       CVE-2015-3397
+branches:
+    master:
```

Comment:
there is no branch with that name... Can you explain a bit more what this is used for?
Seems kind of redundant with the version constraint.